includes/admin/forms/class-metabox-form-data.php

```diff
@@ -225,12 +225,11 @@ function get_settings() {
 								'placeholder' => esc_attr__( 'Give a Custom Amount', 'give' ),
 							),
 						),
-
 						array(
 							'name'		  => 'donation_options_docs',
 							'type'        => 'docs_link',
 							'url'		  => 'https://givewp.com/documentation/core/give-forms/',
-							'title'		  => "Documentation Options",
+							'title'		  => 'Documentation Options',
```

Comment:
All `'title'` fields need to be translatable... Donation Options, Form Display, Donation Goal, etc.